### PR TITLE
Pin mpi4py and vsc-base.

### PR DESCRIPTION
**Pull request by ehiggs**
Pinning vsc-base and mpi4py in its own PR.

### CODE REVIEW

**Comment by boegel** [comment]
@ehiggs: please merge https://github.com/ehiggs/hanythingondemand/pull/20

**Comment by ehiggs** [comment]
@boegel ehiggs#20 merged.

File: setup.py [hunk 1]
```diff
@@ -73,9 +73,14 @@ def find_files(*dirs):
         'Programming Language :: Python :: 2'
     ],
     'install_requires': [
-        'vsc-base >= 1.7.3',
+        # stick with vsc-base 2.2.2 for now;
+        # last tagged release before split between vsc-base and vsc-install causing problems
+        # note: vsc-base version is also pinned in setup.py!
+        'vsc-base == 2.2.2',
```
Comment (boegel) [inline setup.py]: pls bump to 2.4.2

File: tox.ini [hunk 2]
```diff
@@ -3,10 +3,14 @@ envlist=py{26,27}
 
 [testenv]
 deps =
-    git+https://github.com/hpcugent/vsc-install.git
-    git+https://github.com/hpcugent/vsc-base.git
+    # stick with vsc-base 2.2.2 for now;
+    # last tagged release before split between vsc-base and vsc-install causing problems
+    # note: vsc-base version is also pinned in setup.py!
+    git+https://github.com/hpcugent/vsc-base.git@vsc-base-2.2.2
```
Comment (boegel) [inline tox.ini]: pls bump to 2.4.2

**Pull request merged**